target/linux/ramips/base-files/etc/diag.sh

```diff
@@ -231,6 +231,7 @@ get_status_led() {
 	wsr-600)
 		status_led="$board:green:power"
 		;;
+	ex3700|\
```

Comment:
Please move the whole block a few lines up so that the file is in alphabetical order again.